core/src/display_object/stage.rs

```diff
@@ -103,6 +103,9 @@ pub struct StageData<'gc> {
     /// Whether to prevent movies from the changing the stage alignment
     forced_align: bool,
 
+    /// Whether to allow the stage's displayState to be changed.
+    allow_full_screen: bool,
```

Comment:
I think we use `fullscreen` more than `full_screen`.

Reply:
Then I need a `#[serde(rename)]` in `web/src/lib.rs`, since the name of the embed/object attribute is `allowFullScreen`, and the default way `serde` converts camelCase to snake_case makes this `allow_full_screen`.

Reply:
But we don't need to follow the `param` name, right? We already use `backgroundColor` instead of `bgColor`. Maybe it's not worth it.

Reply:
Or we keep the name for loadOptions as well and just rename those internal ones?

Reply:
True, I looked at how "bgcolor" does it. I didn't think of changing the name right after `getOptionString("allowFullScreen");` https://github.com/ruffle-rs/ruffle/blob/e0f5b8906dc1893416d9d836fd4458c8acc2817b/web/packages/core/src/ruffle-player.ts#L2483

Reply:
Changed the functions.